app/src/main/res/layout/activity_multimeter.xml

```diff
@@ -0,0 +1,478 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    app:layout_behavior="@string/appbar_scrolling_view_behavior">
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <TextView
+            android:id="@+id/tv_control_read1"
+            android:layout_width="match_parent"
+            android:layout_height="@dimen/multimeter_display_height"
+            android:layout_gravity="center"
+            android:layout_marginLeft="@dimen/multimeter_display_margin"
+            android:layout_marginRight="@dimen/multimeter_display_margin"
+            android:layout_marginTop="@dimen/multimeter_display_margin_top"
+            android:background="@drawable/tv_border"
+            android:gravity="center"
+            android:textStyle="bold" />
+
+        <TextView
+            android:id="@+id/quantity"
+            android:layout_width="wrap_content"
+            android:layout_height="@dimen/multimeter_display_text_margin"
+            android:layout_alignParentRight="true"
+            android:layout_marginRight="@dimen/multimeter_layout_margin"
+            android:layout_marginTop="@dimen/multimeter_display_text_margin"
+            android:background="@android:color/white"
+            android:fontFamily="sans-serif"
+            android:gravity="center"
+            android:text="200"
```

Comment:
Is this a test line. This is not needed is it?

Reply:
Yes, it is a test line, will remove it :+1: